scripts/update-rarities.sh

```diff
@@ -28,15 +28,11 @@ MOUNTSRARITY_RESPONSE=$(get "$MOUNTSRARITY_SOURCE")
 MOUNTSRARITY=$(echo "$MOUNTSRARITY_RESPONSE" | jq '.mounts | to_entries')
 echo "Downloaded $(echo "$MOUNTSRARITY" | jq 'length') mounts."
 
+sed -i '/Everything after this line/q' MountsRarity.lua
```

Comment:
This should use the variable.

```suggestion
sed -i '/Everything after this line/q' $MOUNTS_FILE
```